app/views/outreach_ai/index.html.slim

```diff
@@ -0,0 +1,52 @@
+div[class="relative overflow-x-auto"]
+  = form_tag outreach_ai_index_path, method: :post, class: "flex flex-col" do
+    div[class="grid grid-cols-2 gap-4 mt-6"]
+      div[class="bg-gray-200"]
+        div[class="bg-white m-4 p-6 pb-8"]
+          h2[class="mb-4"] Outreach Type 
+          ul[class="flex"]
+            - ["Invitation", "New Appeal", "Thank You", "Update"].map do |name|
+              li[data-controller="select-sibling-input"]
+                input[type="radio" name="outreach_ai[outreach_type][name]" value="#{name.parameterize.underscore}" class="hidden peer" data-select-sibling-input-target="radio"]
+                = link_to "#{name}", new_outreach_ai_path(outreach_type: "#{name.parameterize.underscore}"), \
+                  data: { turbo_frame: "outreach_type", action: "click->select-sibling-input#selectRadio" }, \
+                  class: "text-center text-white bg-gray-600 hover:bg-gray-800 focus:ring-4 focus:ring-gray-300 font-medium rounded-lg text-sm px-5 py-2.5 mr-2 mb-2 cursor-pointer peer-checked:bg-amber-600 peer-checked:hover:bg-amber-800"
+          div[class="mt-4"]
+            = turbo_frame_tag "outreach_type"
+        div[class="bg-white m-4 p-6 pb-8"]
+          h2[class="mb-4"] Target Audience 
+          ul[class="flex"]
+            - ["LYBUNT", "Alumni", "Parent", "Student"].map do |name|
+              li
+                input[type="radio" name="outreach_ai[audience]" value="#{name.downcase}" id="#{name.downcase}" class="hidden peer"]
+                label[for="#{name.downcase}" class="text-white bg-gray-600 hover:bg-gray-800 focus:ring-4 focus:ring-gray-300 font-medium rounded-lg text-sm px-5 py-2.5 mr-2 mb-2 cursor-pointer peer-checked:bg-amber-600 peer-checked:hover:bg-amber-800"]
+                  | #{name}
```

Comment:
We don't need to keep track of state when we can just use radio buttons made all fancy with Tailwind.

However, this does depend on Tailwind 3 with their [peer-{modifier} functionality](https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-sibling-state)

Reply:
Without Tailwind 3, we can create this same thing ourselves with some CSS.